packages/components/tools/templates.js

```diff
@@ -85,7 +85,10 @@ const loadContents = glob =>
     contents.forEach((content, templateName) => {
       Handlebars.registerPartial(templateName, content);
       var uids = [];
-      Handlebars.registerHelper('uid', function() {
+      Handlebars.registerHelper('uid', function(repeatLastUid) {
+        if (repeatLastUid === 'repeat') {
```

Comment:
I'd suggest not to do "give me the last one" approach, given tends to be brittle considering distant id-ref source/target. To make my suggestion clearer a bit, I'm suggesting that we set a unique ID in `.config.js`.

Reply:
Oh I see now. So would that be a unique id for the table, and then form the `for` here with that id + row + column?

Reply:
Right. Please note that row has an ID and column has unique name (distinct from displayed title).

Reply:
Got it, but not all rows had an ID, so I used its index as the row identifier: https://github.com/carbon-design-system/carbon/pull/2992/commits/1a220bbc1279ef57ec6a0e5e90b3cf8235e4e52d#diff-99c667b13a41db61508c656d9474e9c7R251